# Define all possible actions as constants here:
uint8 STOP=0
uint8 START=1
uint8 RESTART=2

# And here service itself:
uint8 action
geometry_msgs/PoseStamped[] waypoint_track
---
bool success
